apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/[id]/form.tsx

```diff
@@ -250,6 +253,49 @@ export default function Form({
     }
   }
 
+  async function getSubCompaniesInformation(totalCount = 1000) {
+    try {
+      const response = await fetch(
+        getBaseLink(`/api/crm/subcompanies?maxResultCount=${totalCount}`),
+      );
+
+      if (response.ok) {
+        const _data =
+          (await response.json()) as Volo_Abp_Application_Dtos_PagedResultDto_18;
+        if (_data.items) {
+          setData(_data.items);
+        }
+      } else {
+        toast.error("Failed to fetch Sub Companies.");
+      }
```

Comment:
```suggestion
      if (!response.ok) {
       toast.error(response.statusText);
      }
      
     const _data =
          (await response.json()) as Volo_Abp_Application_Dtos_PagedResultDto_18;
     if (_data.items) {
          setData(_data.items);
        }
```

An easier to read pattern.